advanced-creation.md

```diff
@@ -111,3 +112,119 @@ const unicorn = got.create(settings);
 // Same as:
 const unicorn = got.extend({headers: {unicorn: 'rainbow'}});
 ```
+
+### Merging instances
+
+#### got.mergeInstances(instanceA, instanceB, ...)
+
+Merges many instances into a single one:
+- options are merged using [`got.mergeOptions()`](readme.md#gotmergeoptionsparentoptions-newoptions) (+ hooks are merged too),
+- handlers are stored in an array.
+
+## Usage
+
+#### Denying redirects that lead to other sites than specified
+
+```js
+const controlRedirects = got.create({
+	options: got.defaults.options,
+	handler: (options, next) => {
+		const promiseOrStream = next(options);
+		return promiseOrStream.on('redirect', resp => {
+			const host = new URL(resp.url).host;
+			if (options.allowedHosts && !options.allowedHosts.includes(host)) {
+				promiseOrStream.cancel(`Redirection to ${host} is not allowed`);
+			}
+		});
+	}
+});
+```
+
+#### Limiting download & upload (in case your machine's got a little amount of RAM)
```

Comment:
`(in case your machine's got a little amount of RAM)` should be in text below the title.